themes/default/content/docs/intro/cloud-providers/azure/version-guide.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Azure-Native Version Guide
+meta_desc: How Azure API versions are represented in the native Azure provider for Pulumi.
+---
+
+The native Azure provider SDKs follow the semantic versioning convention, similarly to other Pulumi providers. At the same time, Azure API is structured around date-based API versions defined per Azure resource provider.
+
+This guide describes how we combine both versioning approaches to achieve the following high-level design goals:
+
+- Provide access to the entire API surface of Azure resources.
+- Ship new updates as soon as Microsoft publishes them.
+- Provide stable SDKs without unexpected breaking changes.
+- Enable user experience similar to other Pulumi providers.
+
+## Azure-Native Semantic Versioning
+
+Releases of the Azure-Native SDKs follow the semantic versioning convention, where each version is identified with three numbers: `major.minor.patch`, for example, `1.2.1`:
```

Comment:
Are our criteria for what goes in major, minor, and patch different from the standard semver definitions? If not, then we should just a link to https://semver.org/ and remove the bulleted list. Something like "Releases of the Azure-Native SDKs follow the [semantic versioning convention](https://semver.org/)..."


```suggestion
Releases of the Azure-Native SDKs follow the semantic versioning convention, where each version is identified with three numbers: `major.minor.patch` (for example, `1.2.1`).
```